yaqc_bluesky/_is_discrete.py

```diff
@@ -0,0 +1,31 @@
+from collections import OrderedDict
+import time
+import warnings
+
+from ._has_position import HasPosition
+
+
+class IsDiscrete(HasPosition):
+
+    def _describe(self, out):
+        out = super()._describe(out)
+        meta = OrderedDict()
+        meta["shape"] = []
+        meta["dtype"] = "string"
```

Comment:
The identifier can be null, not sure if that can be declared here or needs to be handled, e.g. by mapping None to empty string

Reply:
`None` works just fine, with the exception of formatting for best effort callback. The messages get emitted and data gets recorded.

I guess now we have to decide, but I think keeping it as-is makes sense.